src/base/QXmppSasl.cpp

```diff
@@ -924,9 +925,9 @@ QByteArray QXmppSaslDigestMd5::serializeMessage(const QMap<QByteArray, QByteArra
             }
         }
         if (quote) {
-            value.replace("\\", "\\\\");
-            value.replace("\"", "\\\"");
-            ba.append("\"" + value + "\"");
+            value.replace(BL("\\"), BL("\\\\"));
```

Comment:
This might need some more testing, not sure how those escape characters end up after a QStringLiterals

Reply:
This shouldn't be a problem. Before the characters were converted exactly the same way, but on runtime.